### ISSUE TITLE
Cannot use Flurry with Xcode 13.3

### ISSUE THREAD

**Issue by eriksargent** [flurry/FlurrySwiftPackage#9]
Flurry cannot be linked into Xcode 13.3 through the Swift Package. It always returns this error:

![image](https://user-images.githubusercontent.com/10200525/156035062-4439b0cf-50c5-4d94-9217-6e7b7b8c1644.png)

I have tried removing flurry from the project and readding it, as well as removing the package and linking it in locally. I have also reset package caches, cleaned the project, and removed all derived data, but this error always happens when I try to use Flurry.

**Comment by rsimpson9292** [comment]
Almost same issue here with Xcode 13.3 **Artifact not found for target 'FlurryConfigBinary'**

**Comment by paulja** [comment]
+1 same issues above

**Comment by ErikLuimes** [comment]
As a temp fix we created a local package and renamed the .binaryTarget to the actual name of the framework. This seems to fix the problem. I also removed redundant use of `.target()`. 

`.testTarget` also caused a problem when building on M1 but can't be bothered to look into it so I removed them.

```
import PackageDescription

let package = Package(
    name: "Flurry",
    platforms: [
        .iOS(.v9),
        .tvOS(.v9),
        .watchOS(.v2)
        ],
    products: [
        // Products define the executables and libraries produced by a package, and make them visible to other packages.
        .library(
            name: "Flurry",
            targets: ["Flurry","CrashReporter"]
        ),
        .library(
            name: "FlurryMessaging",
            targets: ["FlurryMessaging"]
        ),
        .library(
            name: "FlurryConfig",
            targets: ["FlurryConfig"]
        ),
    ],
    dependencies: [
        // Dependencies declare other packages that this package depends on.
        // .package(url: /* package url */, from: "1.0.0"),
    ],
    targets: [
        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
        // Targets can depend on other targets in this package, and on products in packages which this package depends on.
        .binaryTarget(
            name: "Flurry",
            path: "artifacts/Flurry.xcframework"
        ),
        .binaryTarget(
            name: "FlurryMessaging",
            path: "artifacts/FlurryMessaging.xcframework"
        ),
        .binaryTarget(
            name: "FlurryConfig",
            path: "artifacts/FlurryConfig.xcframework"
        ),
        .binaryTarget(
            name: "CrashReporter",
            path: "artifacts/CrashReporter.xcframework"
        )
    ]
)

**Comment by eriksargent** [comment]
> As a temp fix we created a local package and renamed the .binaryTarget to the actual name of the framework. This seems to fix the problem. I also removed redundant use of .target().

Yep that fixes it for me, thanks for the work around!

**Comment by paulja** [comment]
> > As a temp fix we created a local package and renamed the .binaryTarget to the actual name of the framework. This seems to fix the problem. I also removed redundant use of .target().
> 
> Yep that fixes it for me, thanks for the work around!

I'm struggling to get that to work. Any chance you could knock up a quick step list please? Just so I'm not missing anything stupid 🙄

**Comment by vCrespoP** [comment]
+1 here as well.

It would be great to get a PR or a fork for Xcode 13.3 that includes what @eriksargent wrote. Good catch!

**Comment by eriksargent** [comment]
Good idea, here's a patched version you can use directly https://github.com/eriksargent/FlurrySwiftPackage

**Comment by rollyceballos** [comment]
For some reason I tried @eriksargent  fix yesterday and it worked, but it stopped working today.  It now complains it can't find 'Flurry'. I'm confused now 🥺

**Comment by rublagar** [comment]
+1, same issue here.

**Comment by paulja** [comment]
Does anyone know if there are plans to fix this incompatibility with Xcode 13.3? Or at least know what the actual issue is with this package and Xcode? The work arounds here are great, but they are work arounds and they not the whole solution, as they still cause issues/errors, which is obviously sub optimal 😭 

Sadly, of all the packages we use in our Apps this one causes us the most amount of grief—it's stopping my team and I moving to Xcode 13.3. Anyone know any good alternatives to Flurry?

**Comment by SarannyaSalim** [comment]
+1, Can't move to Xcode 13.3 due to this issue.

**Comment by MilesHopkins** [comment]
+1 This is a killer atm

**Comment by flurrydev** [comment]
A patch has bee released as Flurry v11.4.1. Please note only the Package.swift has changed.
